core/modules/ckeditor/ckeditor.module

```diff
@@ -31,6 +31,7 @@ function ckeditor_editor_info() {
   $editors['ckeditor'] = array(
     'label' => t('CKEditor 4'),
     'library' => array('ckeditor', 'backdrop.ckeditor'),
+    'library_version' => CKEDITOR_VERSION,
```

Comment:
Sorry, I see why this is needed now. This can't be pulled from `hook_library_info()` because the library specified is the _Backdrop Library for CKEditor_, not the CKEditor 5 library itself (which is a dependency of the Backdrop Library). I can't think of a better way to get this value, so this might be fine after all.